src/index.js

```diff
@@ -177,6 +182,16 @@ if ( !! window.wcCalypsoBridge.isEcommercePlan ) {
 					}
 					return page;
 				} );
+
+				pages.push( {
+					container: Plugins,
+					path: '/plugins-upgrade',
+					breadcrumbs: [ __( 'Plugins' ), __( 'Plugins' ) ],
```

Comment:
Missing translation domain

```suggestion
					breadcrumbs: [ __( 'Plugins', 'wc-calypso-bridge' ), __( 'Plugins', 'wc-calypso-bridge' ) ],
```